pallets/livingassets-ownership/src/lib.rs

```diff
@@ -63,17 +65,26 @@ pub mod pallet {
         #[pallet::weight(10_000 + T::DbWeight::get().writes(1).ref_time())] // TODO set proper weight
         pub fn create_collection(origin: OriginFor<T>, collection_id: u64) -> DispatchResult {
             let who = ensure_signed(origin)?;
+            Self::do_create_collection(collection_id, who)
+        }
+    }
 
-            ensure!(
-                !OwnerOfCollection::<T>::contains_key(collection_id),
-                Error::<T>::CollectionAlreadyExists
-            );
+    /// LivingAssetsOwnership trait:w
+    pub trait LivingAssetsOwnership<AccountId> {
+        /// Get owner of collection
+        fn owner_of_collection(collection_id: u64) -> Option<AccountId>;
 
-            OwnerOfCollection::<T>::insert(collection_id, &who);
+        /// Create collection
+        fn create_collection(collection_id: u64, who: AccountId) -> DispatchResult;
```

Comment:
in the future, it would be better to have a `CollectionId` type in the pallet config, instead of directly using `u64`. It would help unifying types and avoiding type confusions.